src/App.js

```diff
@@ -25,6 +26,13 @@ function App() {
     const readPost = (id) => {
         console.log("readPost", id)
     }
+    useEffect(() => {
+        const loggedIn = localStorage.getItem("currentUser")
+        if(loggedIn) {
+          setCurrentUser(JSON.parse(loggedIn))
+        }
+        readPost()
+    }, [])
```

Comment:
The useEffect helps when you are performing the fetch requests. You can keep it. Just making sure that you know that it is not serving much of a purpose with the mock data that is provided on the react side.